bin/metrics-vg-usage.rb

```diff
@@ -1,5 +1,6 @@
 #! /usr/bin/env ruby
 #  encoding: UTF-8
+
```

Comment:
Rubocop "Add an empty line after magic comments":
http://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Layout/EmptyLineAfterMagicComment
Some repos adding empty line too. Do I need to ignore it?